modules/tile-converter/src/i3s-converter/helpers/i3s-attribute-transformation.ts

```diff
@@ -0,0 +1,42 @@
+import type {B3DMContent} from '@loaders.gl/3d-tiles';
+import type {ConvertedAttributes} from '../types';
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+import {Writer, WriterOptions} from '@loaders.gl/loader-utils';
+import {convertAttributes} from './geometry-converter';
+
+type AttributesOptions = {
+  useCartesianPostions: boolean;
+};
+
+export type AttributesTransformationOptions = WriterOptions & {
+  attributesTransformations?: AttributesOptions & {
+    useCartesianPostions: false;
+  };
+};
+
+/**
+ * Transform B3DM data to I3S attributes
+ */
+export const I3SAttributeTransformation: Writer = {
```

Comment:
I don't think it is a writer. It should be just worker like in `compression` or `crypto` modules:
"modules/compression/src/compression-worker.ts"

Reply:
Done